src/features/game/expansion/components/temperateSeason/SeasonChanged.tsx

```diff
@@ -0,0 +1,310 @@
+import { Label } from "components/ui/Label";
+import { SUNNYSIDE } from "assets/sunnyside";
+import { Button } from "components/ui/Button";
+import { Context } from "features/game/GameProvider";
+
+import winterBanner from "assets/temperate_seasons/winter_banner.webp";
+import { ITEM_DETAILS } from "features/game/types/images";
+import {
+  InventoryItemName,
+  TemperateSeasonName,
+} from "features/game/types/game";
+import { useSelector } from "@xstate/react";
+import { MachineState } from "features/game/lib/gameMachine";
+import {
+  getHasReadTemperateSeasonTutorial,
+  setHasReadTemperateSeasonTutorial,
+  setLastTemperateSeasonStartedAt,
+} from "features/game/lib/temperateSeason";
+import { SeasonsIntroduction } from "./SeasonsIntroduction";
+import { useTranslation } from "react-i18next";
+import { IngredientsPopover } from "components/ui/IngredientsPopover";
+import { ModalOverlay } from "components/ui/ModalOverlay";
+import { getRelativeTime } from "lib/utils/time";
+
+const SEASON_DETAILS: Record<
+  TemperateSeasonName,
+  {
+    icon: string;
+    inSeason: InventoryItemName[];
+  }
+> = {
+  spring: {
+    icon: SUNNYSIDE.resource.tree_stump,
+    inSeason: [
+      "Sunflower",
+      // "Rhubarb",
+      "Carrot",
+      "Cabbage",
+      "Beetroot",
+      "Parsnip",
+      "Corn",
+      "Radish",
+      "Wheat",
+      "Barley",
+    ],
+  },
+  summer: {
+    icon: SUNNYSIDE.resource.tree_stump,
+    inSeason: [
+      "Sunflower",
+      "Potato",
+      // "Zucchini",
+      "Soybean",
+      // "Hot Pepper",
+      // "Coffee",
+      "Eggplant",
+      "Corn",
+      "Radish",
+      "Wheat",
+    ],
+  },
+  autumn: {
+    icon: SUNNYSIDE.resource.tree_stump,
+    inSeason: [
+      "Sunflower",
+      "Potato",
+      "Pumpkin",
+      "Carrot",
+      // "Broccoli",
+      "Beetroot",
+      "Eggplant",
+      "Wheat",
+      // "Artichoke",
+      "Barley",
+    ],
+  },
+  winter: {
+    icon: SUNNYSIDE.resource.tree_stump,
+    inSeason: [
+      "Sunflower",
+      "Potato",
+      // "Yam",
+      "Cabbage",
+      "Beetroot",
+      "Cauliflower",
+      "Parsnip",
+      // "Onion",
+      // "Turnip",
+      "Wheat",
+      "Kale",
+    ],
+  },
+};
+
+type SeasonEventName = "Tornado" | "Tsunami";
+const DUMMY_EVENT_DATA: Record<
+  SeasonEventName,
+  {
+    icon: string;
+    description: string;
+  }
+> = {
+  Tornado: {
+    icon: SUNNYSIDE.icons.plant,
+    description:
+      "A destructive tornado is approaching! Construct wind turbines to dissipate its energy.",
+  },
+  Tsunami: {
+    icon: SUNNYSIDE.icons.water,
+    description:
+      "A massive wave threatens to flood your crops! Build mangroves along the coast to protect your farm.",
+  },
+};
+
+type WeekData = {
+  day: string;
+  event: SeasonEventName;
+};
+
+const DUMMY_WEEK_DATA: WeekData[] = [
+  { day: "Monday", event: "Tornado" },
+  { day: "Tuesday", event: "Tsunami" },
```

Comment:
Players will not know what the upcoming event will be - only once passed the date, will it reveal which event it was.

In this case I would suggest showing a special event icon (lightning bolt).

The modal could explain that this is a special event: Anything could happen!

- Tsnumi - half crops destroyed
- Double delivery - double delivery rewards
- Insect plague...